src/web/mjs/connectors/oremangaNet.mjs

```diff
@@ -0,0 +1,12 @@
+import oremangaNet from './templates/oremangaNet.mjs';
+
+export default class oremanga extends oremangaNet {
+
+    constructor() {
+        super();
+        super.id = 'oremanga';
+        super.label = 'oremanga';
```

Comment:
label is not necessarly  identical to the id :) You could have written something like "Oremanga", or "OreManga".